etc/adminhtml/system.xml

```diff
@@ -32,7 +32,7 @@
                       <field id="active">1</field>
                     </depends>
                 </field>
-                <field id="token" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="40" translate="label" type="text">
+                <field id="token" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="40" translate="label" type="password">
```

Comment:
Should we also add `<token backend_model="Magento\Config\Model\Config\Backend\Encrypted" />` to our config.xml?
Similarly with `shared_secret` below

Reply:
Good catch - I'd set them via n98 --encrypt so this didn't come up during testing.